spec/integration/post_index_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+RSpec.describe 'User posts', type: :feature do
+  describe 'Show Page' do
+    before(:each) do
+      @user1 = User.create(name: 'Usom', photo: 'https://source.unsplash.com/user/c_v_r/100x100', bio: 'Usom bio')
+      @user2 = User.create(name: 'Burger', photo: 'https://i.picsum.photos/id/216/200/300.jpg?hmac', bio: 'Burger bio')
```

Comment:
##### Dear Team, for a post_index route some tests are missing please consider adding them. 
- [ ] Tests that asserts you can see the number of posts the user has written.
- [ ] Tests that asserts you can see a post's title.